rules-reviewed/eap6/weblogic/weblogic-ignore-references.windup.xml

```diff
@@ -1,31 +0,0 @@
-<?xml version="1.0"?>
-<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="weblogic-ignore-references" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
-
-    <metadata>
-        <description>
-            This ruleset provides references within WebLogic classes implying the class should not be scanned by windup.
-        </description>
-        <dependencies>
-            <addon id="org.jboss.windup.rules,windup-rules-java,2.3.1.Final" />
-        </dependencies>
-        <sourceTechnology id="weblogic" />
-        <targetTechnology id="eap" versionRange="[6,)" />
-        <tag>reviewed-2015-09-28</tag>
-    </metadata>
-    <rules>
-         <javaclass-ignore reference-prefix="weblogic.ejb.container.interfaces.WLEnterpriseBean" />
-         <javaclass-ignore reference-prefix="weblogic.ejb.container.internal" />
-         <javaclass-ignore reference-prefix="weblogic.ejb.container.persistence.spi.EloWrapper"/>
-         <javaclass-ignore reference-prefix="weblogic.ejb20.interfaces"/>
-         <javaclass-ignore reference-prefix="weblogic.ejb20.internal"/>
-         <javaclass-ignore reference-prefix="weblogic.ejbgen"/>
-         <javaclass-ignore reference-prefix="weblogic.ejb20.persistence.spi.CMPBean"/>
-         <javaclass-ignore reference-prefix="weblogic.ejb20.persistence.spi.EoWrapper"/>
-         <javaclass-ignore reference-prefix="weblogic.wsee.tools.clientgen"/>
-         <javaclass-ignore reference-prefix="weblogic.rmi.internal"/>
-         <javaclass-ignore reference-prefix="weblogic.utils.PlatformConstants"/>
-         <javaclass-ignore reference-prefix="weblogic.webservice.core.rpc"/>
-         <javaclass-ignore reference-prefix="oracle.jbo"/>
```

Comment:
I think this will cause us to highlight a lot of generated code. I'm not sure if this is good for the general case?